### DISCUSSION TITLE
Do Faceswap really works in Fooocus, If we Include any image which is not created in Fooocus?

### DISCUSSION THREAD

**Discussion by Shahfahad7866** [lllyasviel/Fooocus#3232] [Q&A] [answered]
Hello friends, Does Faceswap work in Fooocus, If we Include any image that is not created in Fooocus? I tried the image that was generated in Forge and also tried some face images downloaded from the Internet, but it never gave me results. It does not swap faces in the final image at all.

**Comment by mashb1t** [comment, marked as answer]
Yes, works with any image. Make sure to select FaceSwap in the image input > advanced mode.

**Reply by Shahfahad7866** [reply]
I tried, but no luck :(

**Reply by mashb1t** [reply]
please open an issue and provide screenshots / console logs.
